shared/src/config_proxy.rs

```diff
@@ -71,27 +72,33 @@ pub struct CliArgs {
 
 pub const APP_PREFIX: &str = "APP";
 
-/// Parses API-Keys from the environment, expecting:
+/// Parses legacy API-Keys from the environment:
 /// APP_0_ID=app1
 /// APP_0_KEY=App1Secret
 /// APP_1_ID=app2
 /// APP_1_KEY=App2Secret
```

Comment:
Let's now have legacy API keys but instead have a new major version and a simultaneous Bridgehead update.

Reply:
So should I just remove the comment about them or also change that an app identifier is not allowed to start with a number so that all legacy api key entries will lead to errors?

Reply:
Why is it desirable to remove an already existing backwards compatibility? It could update all Bridgeheads simultaneously regardless, but it would not break other systems.

Reply:
> So should I just remove the comment about them or also change that an app identifier is not allowed to start with a number so that all legacy api key entries will lead to errors?

Only remove the comment. Numbers are valid app identifiers. So `APP_0_KEY=secret` would be considered valid. However, the app would then be called `0`.

I hope this also addresses @TKussel 's comment

Reply:
This fully addresses my comment. Thanks.